cmd/promxy/main.go

```diff
@@ -74,6 +74,8 @@ type cliOpts struct {
 	WebCORSOriginRegex string        `long:"web.cors.origin" description:"Regex for CORS origin. It is fully anchored." default:".*"`
 	WebReadTimeout     time.Duration `long:"web.read-timeout" description:"Maximum duration before timing out read of the request, and closing idle connections." default:"5m"`
 
+	MetricsPath string `long:"metrics-path" description:"url path" default:"/metrics"`
```

Comment:
nit: the description here should specify that this is the path for the prometheus metrics endpoint.

Reply:
My latest change should address that :)